ra/ra_test.go

```diff
@@ -996,12 +962,27 @@ func TestPerformValidationNewRPC(t *testing.T) {
 	// Sleep so the RA has a chance to write to the SA
 	time.Sleep(100 * time.Millisecond)
 
-	dbAuthz, err := sa.GetAuthorization(ctx, authz.ID)
-	test.AssertNotError(t, err, "Could not fetch authorization from database")
+	var dbAuthz core.Authorization
+	if features.Enabled(features.NewAuthorizationSchema) {
+		idInt, err := strconv.ParseInt(authz.ID, 10, 64)
```

Comment:
This stanza is repeated a bunch of times. Please extract a helper function that takes as input an authz ID string, checks the feature flag and returns the result of the appropriate call.